M2/Macaulay2/m2/installPackage.m2

```diff
@@ -438,25 +438,27 @@ reproduciblePaths = outf -> (
      outstr := get outf;
      if topSrcdir === null then return outstr;
      srcdir := regexQuote toAbsolutePath topSrcdir;
-     builddir := regexQuote prefixDirectory;
+     prefixdir := regexQuote prefixDirectory;
+     builddir := substring(0, #prefixdir - 9, prefixdir); -- 9 = #"usr-dist/"
```

Comment:
(edit: moved the comment to the PR rather than the commit)
`prefixDirectory` only ends in `usr-dist` when M2 is run from a build directory. Otherwise it's `/usr` or `/usr/local`.

Reply:
Correct, but we won't get this far in the code if we're not running it from the build directory since `topSrcdir` will be `null`, so we'll leave in line 439.

Reply:
Still, something like `replace("usr-dist/?", "", s)` would be safer.

Reply:
Good point!  I was emulating the corresponding code in `startup.m2.in`:
https://github.com/Macaulay2/M2/blob/e78fd58f92763fde048a16af40ab2045b0c3cfc0/M2/Macaulay2/m2/startup.m2.in#L332-L336

I'll push a new commit changing it to use `replace` soon.